Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -1,97 +1,124 @@
+    - I have a deep understanding of the technical direction for all teams within my group so I can detect and correct issues early.
+    - I help teams in my group make critical product architecture and implementation decisions that result in better outcomes.
+    - I manage technical risks effectively, and architect to reduce technical risk.
+- Across Octopus:
+    - I have a broad understanding of the Octopus technical strategy and target architecture.
+    - I am an effective custodian for part of the Octopus target architecture.
+    - I assist in maintaining the Octopus technical strategy by contributing helpful ideas and peer review.
+    - I contribute to the engineering standards and principles used across Octopus.
+    - I collaborate with other Principal Software Engineers to ensure we are making cohesive and complimentary decisions that align with the Octopus technical strategy.
 
 <details>
 <summary>Examples</summary>
 
-- I successfully completed complex tasks spanning multiple domains and teams with high impact. 
-- I helped a team reduce complexity and risk through sound architectural thinking.
-- I developed a plan to evolve the architecture of one of the core components of Octopus within my impact radius.
-- I identified a problem that was timely, important, and impactful, and worked closely with product, engineering, and executive stakeholders to articulate and refine a solution and delivery plan.
-- I worked with teams to manage multiple conflicting priorities and helped sequence work resulting in the best collective outcome.
+- I successfully completed complex tasks spanning multiple domains and teams with high impact.
+- I explored ahead of my team(s) and helped build a technical direction to achieve their mission. I did some exploration solo and some exploration with the team. We experimented with different approaches. We discovered risks and accelerators. We updated the plan when we uncovered new risks/accelerators.
+- I helped a team reduce complexity and risk through sound architectural thinking resulting in better outcomes.
+- I developed a plan to evolve the architecture of a particularly difficult/risky/ambigious part of the core components of Octopus iterating in the open with other people using techniques like RFCs and presentations/breakouts at R&D Weekly and RADAR sessions.
+- I worked with teams to manage multiple conflicting priorities, navigated difficult tradeoffs, and helped sequence work resulting in the best collective outcome.
 - I collaborated with another principal engineer to ensure that a feature I was helping a team with would support a feature I knew another team was planning.
+- I developed a section of the Octopus target architecture, which required a thorough understanding of the problem space, proposing potential solutions, and offering a strong and informed opinion on the best solution.
+- I contributed to building blocks or core technologies used by a variety of teams.
```

Comment:
```suggestion
```

This feels more like an indicator that the engineer "is in Foundations Group", as opposed to "is at Principal Engineer" level.

Reply:
I disagree that this is only the domain of foundations group. A good example would be the building blocks Jim has developed for the new executions pipeline. Or the observability plumbing Rob W has been working on. Principals tend to contribute to core technology used across teams. 

That being said, contributing to building blocks / core technologies doesn't feel like an L5-specific concern, so if that was the basis of the reasoning I'd be happy to see it go.

An alternative would be 

`I designed and implemented core technologies used by a variety of teams`, with an emphasis on not being a contributor, but really the creator of these things.

Reply:
> contributing to building blocks / core technologies doesn't feel like an L5-specific concern

Yes, that's the basis of my reasoning 👍

Reply:
I've made an edit using your suggestion @andrewabest.